src/OrbitGl/App.cpp

```diff
@@ -1148,6 +1146,16 @@ void OrbitApp::StartCapture() {
   bool collect_memory_info = data_manager_->collect_memory_info();
   uint64_t memory_sampling_period_ms = data_manager_->memory_sampling_period_ms();
 
+  std::optional<int64_t> memory_information_sampling_period_ms_opt = std::nullopt;
+  if (collect_memory_info) {
+    memory_information_sampling_period_ms_opt = static_cast<int64_t>(memory_sampling_period_ms);
```

Comment:
In `CreateCaptureStartData`, `capture_start_data.memory_information_sampling_period_ms` is set to `-1` if the memory information collection is disabled. And here we have `memory_information_sampling_period_ms_opt` equals nullopt if `collect_memory_info` is false. 

I feel like that it would be better to use a consistent way to indicate that memory information collection is disabled, either using -1 or using nullopt.

Reply:
done

Reply:
I will change it to not use std::optional. Thank you @vickyliu-go4it for the suggestion

Reply:
done